File: finalproject-tayviet-accessory-store-management.admin/src/app/Interface/iaccount.ts
export interface IAccount {
  id: string;
  name: string;
  email: string;
  password: string;
  phoneNumber: string;
  username: string;
  state: string;
  role: string;
  image: string | ArrayBuffer | null;
}

export class Account implements IAccount {
  id: string = "";
  name: string = "";
  email: string = "";
  password: string = "";
  phoneNumber: string = "";
  username: string = "";
  state: string = "";
  role: string = "";
  image: string | ArrayBuffer | null = "";
}

File: FinalProject-TayViet-Accessory-Store-Management.Server/Interfaces/IAccount.cs
namespace FinalProject_TayViet_Accessory_Store_Management.Server.Interfaces
{
    public interface IAccount
    {
        string id { get; set; }
        string name { get; set; }
        string email { get; set; }
        string password { get; set; }
        string phoneNumber { get; set; }
        string username { get; set; }
        string? state { get; set; }
        string? role { get; set; }

        void Login();
        void Logout();
    }
}
